client/components/foldable-card/index.jsx

```diff
@@ -85,10 +86,12 @@ var FoldableCard = React.createClass( {
 			);
 		}
 		if ( this.props.children ) {
-			let iconSize = 24;
+			const iconSize = 24;
+			const screenReaderDefaultText = ( typeof this.translate === 'function' ) ? this.translate( 'More' ) : '';
```

Comment:
I'd like that very much.  I will update this PR to always pass the screen reader text as a prop, including fixing the existing callers to do so.